### PR TITLE
feat: Adds a production deploy

### PR DESCRIPTION
**Pull request by ElvenSpellmaker**
#### 📲 What

Adds a production release for the application (will be gated to master after build passes)

#### 🤔 Why

We need a more stable environment gated behind master.

#### ✅ Acceptance criteria Checklist

- [x] Code peer reviewed?
- [x] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: build/azDevops/azure/azure-pipelines-javaspring-k8s.yml [hunk 1]
```diff
@@ -62,6 +62,7 @@ variables:
   self_repo_src: java
   self_repo_tf_src: deploy/azure/app/kube
   self_generic_name: stacks-api
+  self_pipeline_repo: stacks-pipeline-templates
```
Comment (ElvenSpellmaker) [inline build/azDevops/azure/azure-pipelines-javaspring-k8s.yml:65]: This is relating to another branch, best move it to the other branch before merging.

File: build/azDevops/azure/azure-pipelines-javaspring-k8s.yml [hunk 2]
```diff
@@ -92,6 +93,7 @@ variables:
   # BUILD ARTIFACTS across stages
   build_artifact_deploy_path: $(Agent.BuildDirectory)/s/$(self_repo)/deploy/k8s/app
   build_artifact_deploy_name: $(self_generic_name)
+  build_scripts_directory: "$(Agent.BuildDirectory/s/$(self_pipeline_repo)/scripts"
```
Comment (ElvenSpellmaker) [inline build/azDevops/azure/azure-pipelines-javaspring-k8s.yml:96]: This is relating to another branch, best move it to the other branch before merging.

File: deploy/k8s/app/base_api-deploy.yml [hunk 3]
```diff
@@ -192,6 +192,8 @@ spec:
               value: ${k8s_app_path}
             - name: LOG_LEVEL
               value: debug
+            - name: AZURE_COSMOSDB_URI
```
Comment (peet-j) [inline deploy/k8s/app/base_api-deploy.yml:195]: That should be the one 👍

Comment (ElvenSpellmaker) [reply deploy/k8s/app/base_api-deploy.yml:195]: Perfect!

**Pull request merged**